gtfs/spec/en/reference.md

```diff
@@ -477,12 +479,16 @@ To process the cost of a leg:
 
 <br/>
 
+For area set predicates in `fare_leg_rules.txt` specified below, a leg “travels through an area” if any of the stops or parent stations (if the area is not defined at the stop-level) of the leg; including departure, arrival, and intermediate stops; belongs to the specified area as defined by [stop_areas.txt](#stopareastxt).
```

Comment:
For the intermediate stops, it seems they must be stops defined in stop_times rather than "geographical pass-through" stops. Could the current description cause misunderstandings? Would it be clearer to state this explicitly?

Reply:
I attempted to add explicit language.  Let me know what you think.

Reply:
Looks good to me!

Just to confirm:

> and intermediate stops as referenced by the trip's stop-time entries; belongs to the specified area as defined by [stop_areas.txt]

This means that entries will be included regardless of their `pickup_type` and `drop_off_type` values (e.g., value=1 - No pickup/drop_off available). Is this correct?

Reply:
Yes, I suppose that's correct.

I think we discussed this briefly in the last meeting, yes?  I think it's reasonable to make the default the simple case (passes through a stop).

Reply:
Yes, got it. Just want to confirm if I correctly understand the semantics here (i.e. regardless of the pickup_type and drop_off_type values).